src/Annotations/AppKernel.php

```diff
@@ -54,6 +54,7 @@ public function __construct(string $environment = 'dev')
 
         $this->app = new Application([
             'debug' => $config['debug'] ?? false,
+            'containerized' => $config['containerized'] ?? false,
```

Comment:
is the `$somevar ?? false` syntax the same as `$somevar || false` or `somevar or False` in Python? I remember php had/has the `somecond? true: false` type syntax.

Reply:
it's sugar over `isset()` usage http://php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op which allows for any expression to fall back to a default value